src/page-example/page-rich-text/README.md

```diff
@@ -0,0 +1,7 @@
+# ECL Example page - Home
+
+npm package: `@ecl/page-home`
```

Comment:
for what it counts, since we don't release these packages, this should be updated :)